src/test/java/subway/StationAcceptanceTest.java

```diff
@@ -26,42 +26,84 @@ public class StationAcceptanceTest {
     @DisplayName("지하철역을 생성한다.")
     @Test
     void createStation() {
-        // when
-        Map<String, String> params = new HashMap<>();
-        params.put("name", "강남역");
 
-        ExtractableResponse<Response> response =
-                RestAssured.given().log().all()
-                        .body(params)
-                        .contentType(MediaType.APPLICATION_JSON_VALUE)
-                        .when().post("/stations")
-                        .then().log().all()
-                        .extract();
+        //when
+        ExtractableResponse<Response> response = createSubwayStation("신림역");
+        List<String> stationNames = getStationNames();
 
-        // then
+        //then
         assertThat(response.statusCode()).isEqualTo(HttpStatus.CREATED.value());
-
-        // then
-        List<String> stationNames =
-                RestAssured.given().log().all()
-                        .when().get("/stations")
-                        .then().log().all()
-                        .extract().jsonPath().getList("name", String.class);
-        assertThat(stationNames).containsAnyOf("강남역");
+        assertThat(stationNames).containsAnyOf("신림역");
     }
 
     /**
      * Given 2개의 지하철역을 생성하고
      * When 지하철역 목록을 조회하면
      * Then 2개의 지하철역을 응답 받는다
      */
-    // TODO: 지하철역 목록 조회 인수 테스트 메서드 생성
+    @DisplayName("지하철역을 조회한다")
+    @Test
+    void getStation() {
+
+        //given
+        createSubwayStation("신림역");
+        createSubwayStation("봉천역");
+
+        //when
+        List<String> stationNames = getStationNames();
+
+        //then
+        assertThat(stationNames).containsExactly("신림역", "봉천역");
+
+    }
 
     /**
      * Given 지하철역을 생성하고
      * When 그 지하철역을 삭제하면
      * Then 그 지하철역 목록 조회 시 생성한 역을 찾을 수 없다
      */
-    // TODO: 지하철역 제거 인수 테스트 메서드 생성
+    @DisplayName("지하철역을 삭제한다")
+    @Test
+    void deleteStation() {
+
+        //given
+        long stationId = createSubwayStation("사상역").jsonPath().getLong("id");
+
+        //when
+        ExtractableResponse<Response> response = RestAssured.given().log().all()
+            .contentType(MediaType.APPLICATION_JSON_VALUE)
+            .basePath("/stations")
+            .pathParam("id", stationId)
+            .when().delete("{id}")
+            .then().log().all()
+            .extract();
+        List<String> stationNames = getStationNames();
+
+        //then
+        assertThat(response.statusCode()).isEqualTo(HttpStatus.NO_CONTENT.value());
+        assertThat(stationNames).doesNotContain("사상역");
```

Comment:
```suggestion
        //when
        ExtractableResponse<Response> response = RestAssured.given().log().all()
            .contentType(MediaType.APPLICATION_JSON_VALUE)
            .basePath("/stations")
            .pathParam("id", stationId)
            .when().delete("{id}")
            .then().log().all()
            .extract();
        

        //then
        List<String> stationNames = getStationNames();
        assertThat(response.statusCode()).isEqualTo(HttpStatus.NO_CONTENT.value());
        assertThat(stationNames).doesNotContain("사상역");
```

given / when / then에 대한 플로우를 본다면 위와 같은 순서로 되어야한다고 생각해요.

또한 delete도 create나 getStationNames 같이 메서드로 만들어볼까요?